src/frontend/admin/src/components/templates/orders/buttons/OrderActions.tsx

```diff
@@ -3,6 +3,7 @@ import { useMemo } from "react";
 import { defineMessages, useIntl } from "react-intl";
 import CancelIcon from "@mui/icons-material/Cancel";
 import ArticleIcon from "@mui/icons-material/Article";
+import UndoIcon from "@mui/icons-material/Undo";
```

Comment:
Is the icon clear enough on its purpose? Else I can suggest SettingsBackupRestore or CurrencyExchange

Reply:
CurrencyExchange is nice, thanks !